modin/config/envvars.py

```diff
@@ -628,13 +628,34 @@ class TestReadFromPostgres(EnvironmentVariable, type=bool):
     default = False
 
 
+class ModinGithubCI(EnvironmentVariable, type=bool):
```

Comment:
```suggestion
class GithubCI(EnvironmentVariable, type=bool):
```
We do not have prefixes for configuration classes. Do we really need one here?

Reply:
Fixed!